2021-11-03 10:31:50,790 INFO    MainThread:87385 [wandb_setup.py:_flush():69] setting env: {}
2021-11-03 10:31:50,790 INFO    MainThread:87385 [wandb_setup.py:_flush():69] setting login settings: {}
2021-11-03 10:31:50,790 INFO    MainThread:87385 [wandb_init.py:_log_setup():348] Logging user logs to /home/pholur/Situation_Modeling/wandb/run-20211103_103150-3uoabywu/logs/debug.log
2021-11-03 10:31:50,790 INFO    MainThread:87385 [wandb_init.py:_log_setup():349] Logging internal logs to /home/pholur/Situation_Modeling/wandb/run-20211103_103150-3uoabywu/logs/debug-internal.log
2021-11-03 10:31:50,790 INFO    MainThread:87385 [wandb_init.py:init():381] calling init triggers
2021-11-03 10:31:50,791 INFO    MainThread:87385 [wandb_init.py:init():388] wandb.init called with sweep_config: {}
config: {}
2021-11-03 10:31:50,791 INFO    MainThread:87385 [wandb_init.py:init():430] starting backend
2021-11-03 10:31:50,791 INFO    MainThread:87385 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-11-03 10:31:50,859 INFO    MainThread:87385 [backend.py:ensure_launched():135] starting backend process...
2021-11-03 10:31:50,932 INFO    MainThread:87385 [backend.py:ensure_launched():140] started backend process with pid: 88321
2021-11-03 10:31:50,935 INFO    MainThread:87385 [wandb_init.py:init():435] backend started and connected
2021-11-03 10:31:50,937 INFO    MainThread:87385 [wandb_init.py:init():494] updated telemetry
2021-11-03 10:31:50,938 INFO    MainThread:87385 [wandb_init.py:init():517] communicating current version
2021-11-03 10:31:52,008 INFO    MainThread:87385 [wandb_init.py:init():522] got version response upgrade_message: "wandb version 0.12.6 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-11-03 10:31:52,008 INFO    MainThread:87385 [wandb_init.py:init():530] communicating run to backend with 30 second timeout
2021-11-03 10:31:52,172 INFO    MainThread:87385 [wandb_init.py:init():557] starting run threads in backend
2021-11-03 10:31:57,179 INFO    MainThread:87385 [wandb_run.py:_console_start():1605] atexit reg
2021-11-03 10:31:57,180 INFO    MainThread:87385 [wandb_run.py:_redirect():1479] redirect: SettingsConsole.REDIRECT
2021-11-03 10:31:57,180 INFO    MainThread:87385 [wandb_run.py:_redirect():1484] Redirecting console.
2021-11-03 10:31:57,184 INFO    MainThread:87385 [wandb_run.py:_redirect():1540] Redirects installed.
2021-11-03 10:31:57,184 INFO    MainThread:87385 [wandb_init.py:init():582] run started, returning control to user process
2021-11-03 10:32:04,962 INFO    MainThread:87385 [wandb_run.py:_atexit_cleanup():1575] got exitcode: 1
2021-11-03 10:32:04,963 INFO    MainThread:87385 [wandb_run.py:_restore():1547] restore
2021-11-03 10:32:07,797 INFO    MainThread:87385 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 734
  total_bytes: 734
}

2021-11-03 10:32:08,258 INFO    MainThread:87385 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 734
  total_bytes: 734
}

2021-11-03 10:32:08,412 INFO    MainThread:87385 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 734
  total_bytes: 734
}

2021-11-03 10:32:09,204 INFO    MainThread:87385 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 734
  total_bytes: 734
}

2021-11-03 10:32:09,307 INFO    MainThread:87385 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 734
  total_bytes: 10503
}

2021-11-03 10:32:09,410 INFO    MainThread:87385 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10503
  total_bytes: 10503
}

2021-11-03 10:32:09,514 INFO    MainThread:87385 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10503
  total_bytes: 10503
}

2021-11-03 10:32:09,617 INFO    MainThread:87385 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10503
  total_bytes: 10503
}

2021-11-03 10:32:09,726 INFO    MainThread:87385 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10503
  total_bytes: 10503
}

2021-11-03 10:32:09,830 INFO    MainThread:87385 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10503
  total_bytes: 10503
}

2021-11-03 10:32:09,950 INFO    MainThread:87385 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10503
  total_bytes: 10503
}

2021-11-03 10:32:10,232 INFO    MainThread:87385 [wandb_run.py:_wait_for_finish():1697] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10503
  total_bytes: 10503
}
local_info {
}

2021-11-03 10:32:11,583 INFO    MainThread:87385 [wandb_run.py:_show_files():1947] logging synced files
